client/src/components/avatar/avatar.jsx

```diff
@@ -0,0 +1,13 @@
+import s from './avatar.module.sass';
+
+const Avatar = ({url}) =>{
```

Comment:
url => src
+ add a className prop, so this can be styled from outside

Reply:
I corrected